apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -43,6 +71,54 @@ export const DashboardPage = ({ currentUser }: DashboardPageProps) => {
         )}
         subtitle={""}
       />
+      <section data-h2-margin="base(x3, 0)">
+        <div data-h2-wrapper="base(center, large, x1) p-tablet(center, large, x2)">
+          <div
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column) p-tablet(row)"
+            data-h2-gap="base(x1)"
+          >
+            {isVerifiedEmployee ? (
+              <CareerDevelopmentTaskCard
+                moveInterests={moveInterestsMapped}
+                organizationTypeInterests={organizationTypeInterestsMapped}
+                communityInterestsQuery={communityInterestsQuery}
+              ></CareerDevelopmentTaskCard>
+            ) : null}
+            <div
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(column)"
+              data-h2-gap="base(x1)"
+              data-h2-max-width="p-tablet(x14)"
```

Comment:
I'm not saying this is incorrect but it gives me pause :sweat_smile: We don't use max width a lot so I'm just checking in.

@JoshBeveridge Is this expected or do we have a container with a standardized max width we can use here?

Reply:
I copied the `ManagerDashboardPage` styling given we are repurposing that page

![image](https://github.com/user-attachments/assets/8ea7beea-4f84-457b-8cd2-1ca6c34384d6)

Reply:
@esizer @vd1992 Hmm... this is applied to the sidebar directly, yeah? Gonna get the PR up and running on my machine to take a look.

Reply:
Okay, yeah, this looks good to me - I think the reason we set a value here was because the sidebar on the dashboard (and a handful of other pages coming up) is wider than the standard table of contents sidebar. For optimization purposes, I'm not opposed to moving this value into a container for easier reuse though!